jenkins/Machines.groovy

```diff
@@ -22,6 +22,14 @@ def eiger = [name: 'eiger',
            modulesProduction: '',
            prefixProduction: '/apps/eiger/UES/jenkins/1.3.2/20.10']
 
+def pilatus = [name: 'pilatus',
```

Comment:
Just format the dictionary so that the keys fall in the same column.

Reply:
Thanks @teojgo, does may last commit address your remark?